internal/util/telemetry/telemetry.go

```diff
@@ -60,15 +60,18 @@ func (s *Flag) UnmarshalText(text []byte) error {
 //   - common DO_NOT_TRACK environment variable;
 //   - executable name;
 //   - and the previously saved state.
-func initialState(f *Flag, dnt string, execName string, prev *bool, l *zap.Logger) (*bool, error) {
+//
+// The second returned value is true if the telemetry state should be locked, because of
+// setting telemetry via a command-line flag, an environment variable, or a filename.
+func initialState(f *Flag, dnt string, execName string, prev *bool, l *zap.Logger) (*bool, bool, error) {
```

Comment:
It's quite difficult to follow each return value. Possibly using `var locked bool` and returning its value would improve.

Reply:
Just to clarify, you mean that we could replace every `return ..., false, ...`  with `return ..., locked, ...`, right?

Reply:
Yeah, for me that would make it easier to follow :hugs: